calico/felix/devices.py

```diff
@@ -226,6 +226,30 @@ def interface_up(if_name):
     return bool(int(flags, 16) & 1)
 
 
+def remove_conntrack_flows(ip_addresses):
+    """
+    Removes any conntrack entries that use any of the given IP
+    addresses in their source/destination.
+    """
+    for ip in ip_addresses:
+        _log.debug("Removing conntrack rules for %s", ip)
+        for direction in ["--orig-src", "--orig-dst",
+                          "--reply-src", "--reply-dst"]:
+            try:
+                futils.check_call(["conntrack", "--delete", direction, ip])
+            except FailedSystemCall as e:
```

Comment:
My reading of the conntrack man page is that you need to explicitly specify IPv4 or IPv6 (and that it defaults to IPv4, which is why this will work for IPv4).